src/commands/deploy/nextjs/deploy.ts

```diff
@@ -117,8 +130,8 @@ export async function nextJsDeploy(options: GenezioDeployOptions) {
     );
 }
 
-async function checkProjectLimitations() {
-    const assetsPath = path.join(process.cwd(), ".open-next", "assets");
+async function checkProjectLimitations(cwd?: string) {
```

Comment:
Let's not have this parameter optional as it is confusing. This function is never used without an argument.